keyboards/fractal/config.h

```diff
@@ -0,0 +1,58 @@
+#ifndef CONFIG_H
```

Comment:
Could you replace the first two lines (up to and including the define) with `#pragma once` instead, and remove the `#endif` at the end?